src/components/IconButton/IconButton.tsx

```diff
@@ -42,13 +43,16 @@ export interface IconButtonProps extends VibeComponentProps {
    * Size of the icon
    */
   size?: Size;
-
   /**
    * Whether the tooltip should be displayed or not
    */
   hideTooltip?: boolean;
   /**
-   * Tooltip wraps the button icon, it will display in the tooltip, if not present the aria label will be shown
+   * Props for Tooltip wrapper component
```

Comment:
```suggestion
   * Props for Tooltip component
```

wrapper is redundant